app/dts/bindings/zmk,ext-power-generic.yaml

```
@@ -0,0 +1,20 @@
+#
+# Copyright (c) 2020, The ZMK Contributors
+# SPDX-License-Identifier: MIT
+#
+
+description: |
+  Generic driver for controlling the external power output
+  by toggling the control-gpio pin status
+  (Only in supported hardware)
+
+compatible: "zmk,ext-power-generic"
+
+properties:
+  control-gpios:
+    type: phandle-array
+    required: true
+  label:
+    type: string
+    required: false
```

This should be required, since it's used to register the single driver instance in the driver itself:

```suggestion
    required: true
```